### PR TITLE
fixing "return" issue on function customSaveDataAndFile vente privee

### PR DESCRIPTION
**Pull request by SomeAverageDev**
from issue https://github.com/cozy-labs/konnectors/issues/351

### CODE REVIEW

File: server/konnectors/vente_privee.js [hunk 1]
```diff
@@ -212,39 +213,44 @@ function parsePage(requiredFields, bills, data, next) {
...
-  bills.fetched.length);
+    bills.fetched.length);
   return next();
 }
 
 function customFilterExisting(requiredFields, bills, data, next) {
   log.debug('customFilterExisting');
   filterExisting(log, Bill)(requiredFields, bills, data, next);
-  return next();
+  return true;
```
Comment (ZeHiro) [inline server/konnectors/vente_privee.js]: You should return on filterExisting

File: server/konnectors/vente_privee.js [hunk 2]
```diff
@@ -212,39 +213,44 @@ function parsePage(requiredFields, bills, data, next) {
...
 }
 
 function customSaveDataAndFile(requiredFields, bills, data, next) {
   log.debug('customSaveDataAndFile');
   const fnsave = saveDataAndFile(
     log, Bill, fileOptions, ['vente-privee', 'facture']);
   fnsave(requiredFields, bills, data, next);
-  return next();
```
Comment (ZeHiro) [inline server/konnectors/vente_privee.js]: You should return on fnsave

File: server/konnectors/vente_privee.js [hunk 3]
```diff
@@ -212,39 +213,44 @@ function parsePage(requiredFields, bills, data, next) {
...
+  return true;
 }
 
 function buildNotifContent(requiredFields, bills, data, next) {
   log.debug('buildNotifContent');
   if (bills.filtered.length > 0) {
-    const localizationKey = 'notification bills';
+    const localizationKey = 'notification vente_privee';
```
Comment (ZeHiro) [inline server/konnectors/vente_privee.js]: the notification vente_privee does not exist anymore. You hsould use the 'generic one'

**Comment by ZeHiro** [comment]
@SomeAverageDev did you test it? Can you confirm it wors? Thank you.

**Comment by SomeAverageDev** [comment]
hello @zehiro,
I did, and it worked ;-)

**Pull request merged**